### ISSUE TITLE
Syllabus page or index page to go through the course outline 

### ISSUE THREAD

**Issue by tanvidomadia** [Simon-Initiative/oli-torus#2610]
**Problem related to the feature request**
Currently, I have to click on the next or previous button to navigate the course when I click on preview. This gets tedious if I want to jump to the last page of the course without having to go through all the pages of the units/modules.  

**Solution preferred**
A syllabus or index view of the course structure within the preview and published version of the course that has hyperlinks and navigation to particular course pages 

****Alternatives considered****
- Linkedin learning and many other platforms have a column on the side with the syllabus/ course structure view. 
- On the OLI platform there is the syllabus page.

**Comment by eliknebel** [comment]
Agree, this would be helpful. In the meantime a workaround is to simply preview the page from the curriculum page authoring view

**Comment by eliknebel** [comment]
Project Preview (Publish > Review > Preview) should take you to a preview of the course outline.

**Comment by eliknebel** [comment]
This is more important for large migrated courses
